frontend/packages/knative-plugin/integration-tests/features/kafka-event-source/eventing-kafka-event-source.feature

```diff
@@ -4,8 +4,9 @@ Feature: Kafka Event Source
 
 
         Background:
-            Given Kafka Operator installed
-              And BootStrapServers and Topics created
+            Given user has installed Serverless Operator
+              And user has created kafka instance in knative-eventing namespace
+              And user has created BootStrapServers and Topics
```

Comment:
for RHOAS user needs to do the below but for AMQ streams what you have is correct

```suggestion
              And user has created BootStrapServers and Topics in Kafka cluster
```

Reply:
This file specifically deals with AMQ only

Reply:
Added AMQ stream operator in Background